### PR TITLE
Add LIEnergy device

### PR DESCRIPTION
**Pull request by murilobalves**

### CODE REVIEW

File: siriuspy/siriuspy/devices/lienergy.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+        LI = 'LI-Glob:AP-MeasEnergy'
+        ALL = (LI, )
+
+    _properties = (
+        'Dispersion-SP', 'Dispersion-RB',
+        'IntDipole-Mon', 'Energy-Mon', 'Spread-Mon',
+        'MeasureCtrl-Sel', 'MeasureCtrl-Sts')
+
+    def __init__(self, devname):
```
Comment (fernandohds564) [inline siriuspy/siriuspy/devices/lienergy.py:20]: since there is only one device, I think this argument should at least be optional.

**Review by fernandohds564 — CHANGES_REQUESTED** [review]

File: siriuspy/siriuspy/devices/lienergy.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    class DEVICES:
+        """Devices names."""
+
+        LI = 'LI-Glob:AP-MeasEnergy'
+        ALL = (LI, )
+
+    _properties = (
+        'Dispersion-SP', 'Dispersion-RB',
+        'IntDipole-Mon', 'Energy-Mon', 'Spread-Mon',
+        'MeasureCtrl-Sel', 'MeasureCtrl-Sts')
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/lienergy.py:18]: It seems that is missing `'Spectrometer-RB'`and `'Spectrometer-SP'`properties

Comment (murilobalves) [reply siriuspy/siriuspy/devices/lienergy.py:18]: Initially I included these properties but when I tested, there was not connection with them. This is why I remove it.

Comment (murilobalves) [reply siriuspy/siriuspy/devices/lienergy.py:18]: Initially I included and just removed from _properties, I forgot to remove the properties. Thank you!

**Review by fernandohds564 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Pull request merged**